modules/manually-create-identity-access-management.adoc

```diff
@@ -163,12 +193,14 @@ endif::google-cloud-platform[]
 
 . Create YAML files for secrets in the `openshift-install` manifests directory that you generated previously. The secrets must be stored using the namespace and secret name defined in the `spec.secretRef` for each `CredentialsRequest` object. The format for the secret data varies for each cloud provider.
```

Comment:
I suspect the answer is yes, but want to verify. I noticed that ASH UPI doc that details how to manually create cloud credentials had an additional step (8d)[1] from the material I was working with. Does creating the `cco-configmap.yaml` file also apply to this section?

[1] https://deploy-preview-39366--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_azure_stack_hub/installing-azure-stack-hub-user-infra.html#installation-user-infra-generate-k8s-manifest-ignition_installing-azure-stack-hub-user-infra

Reply:
It looks like this step may actually no longer be required based on my local testing. There may have been a change in the CCO that no longer requires this. I think we are safe to leave it out.